config/Development.toml

```diff
@@ -33,6 +33,7 @@ connection_timeout = 10
 host = ""
 mock_locker = true
 basilisk_host = ""
+locker_setup = "legacy_locker"
```

Comment:
Yes, but I kept it as a configuration because we are exploring the possibility of using basilisk-hs in sandbox as well in the future, having it as a config gives us more control in this case.